plugin/src/test/java/io/jenkins/plugins/casc/SchemaGenerationTest.java

```diff
@@ -1,60 +1,39 @@
 package io.jenkins.plugins.casc;
 
-import com.google.gson.Gson;
-import com.google.gson.GsonBuilder;
-import com.google.gson.JsonElement;
-import com.google.gson.JsonParser;
 import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
 import io.jenkins.plugins.casc.misc.Util;
-import org.everit.json.schema.Schema;
-import org.everit.json.schema.ValidationException;
-import org.everit.json.schema.loader.SchemaLoader;
 import org.json.JSONObject;
 import org.json.JSONTokener;
 import org.junit.Rule;
 import org.junit.Test;
 
-import static io.jenkins.plugins.casc.SchemaGeneration.generateSchema;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 public class SchemaGenerationTest {
+
     @Rule
     public JenkinsConfiguredWithCodeRule j = new JenkinsConfiguredWithCodeRule();
 
     @Test
-    public void validSchemaShouldSucceed() throws Exception {
-        JSONObject schemaObject = generateSchema();
-        JSONObject jsonSchema = new JSONObject(
-            new JSONTokener(schemaObject.toString()));
-        String yamlStringContents = Util.toStringFromYamlFile(this, "validSchemaConfig.yml");
-        JSONObject jsonSubject = new JSONObject(
-            new JSONTokener(Util.convertToJson(yamlStringContents)));
-        Schema schema = SchemaLoader.load(jsonSchema);
-        try {
-            schema.validate(jsonSubject);
-        } catch (Exception e) {
-            fail(e.getMessage());
-        }
+    public void validSchemaShouldSucceed() {
+        assertTrue(Util.validateSchema(returnJSONSubject("validSchemaConfig.yml")));
```

Comment:
thoughts on writing a custom matcher?
https://www.baeldung.com/hamcrest-custom-matchers

It could be:
`isValidSchema()`

at a minimum this should be statically imported:
```suggestion
        assertTrue(validateSchema(returnJSONSubject("validSchemaConfig.yml")));
```